### ISSUE TITLE
The results of the inference are confusing using Open-Sora 1.3 Model Weights

### ISSUE THREAD

**Issue by Gracewx** [hpcaitech/Open-Sora#800]
The results are as follows when I use the t2v and i2v inference commands. I downloaded the Open-Sora 1.3 Model Weights to the local path. 
What's the reason for this?

t2v configs:
num_frames = 49 #113
fps = 24
frame_interval = 1
resolution = "360p" #"720p"
aspect_ratio = "9:16"

save_dir = "./samples/samples/"
multi_resolution = "STDiT2"
seed = 42
batch_size = 1
dtype = "bf16"

model = dict(
    type="STDiT3-XL/2",
    from_pretrained="/workspace/Open-Sora/hpcai-tech/OpenSora-STDiT-v4",
    qk_norm=True,
    enable_flash_attn=True,
    enable_layernorm_kernel=True,
    kernel_size=(8, 8, -1),
    use_spatial_rope=True,
    force_huggingface=True,
)
vae = dict(
    type="OpenSoraVAE_V1_3",
    from_pretrained="/workspace/Open-Sora/hpcai-tech/OpenSora-VAE-v1.3",
    z_channels=16,
    micro_batch_size=1,
    micro_batch_size_2d=4,
    micro_frame_size=17,
    use_tiled_conv3d=True,
    tile_size=4,
    normalization="video",
    temporal_overlap=True,
    force_huggingface=True,
)
text_encoder = dict(
    type="t5",
    from_pretrained="/workspace/Open-Sora/pretrained_models/DeepFloyd/t5-v1_1-xxl",
    model_max_length=300,
)
scheduler = dict(
    type="rflow",
    use_timestep_transform=True,
    num_sampling_steps=30,
    cfg_scale=7.5,
    use_oscillation_guidance=True,
    use_flaw_fix=True,
    # transform_scale=2.0,
)

t2v inference command:
python scripts/inference.py \
configs/opensora-v1-3/inference/t2v.py   \
--num-frames 49 \
--resolution 360p \
--aspect-ratio 9:16   \
--prompt "a beautiful waterfall"

![Image](https://github.com/user-attachments/assets/4d4eb04b-edc1-40af-8b72-f6565ad4e401)

v2v configs:
num_frames = 49 # 113
resolution = "360p"
aspect_ratio = "9:16"
fps = 24
frame_interval = 1

save_dir = "samples/v2v/debug"
multi_resolution = "STDiT2"
seed = 42
batch_size = 1
dtype = "bf16"

cond_type = "i2v_head"  # v2v_head, i2v_head, i2v_loop, or None
condition_frame_length = 5  # condition on the first n frames, latent size
use_sdedit = True
use_oscillation_guidance_for_text = True
use_oscillation_guidance_for_image = True

model = dict(
    type="STDiT3-XL/2",
    from_pretrained="/workspace/Open-Sora/hpcai-tech/OpenSora-STDiT-v4",
    qk_norm=True,
    enable_flash_attn=True,
    enable_layernorm_kernel=True,
    kernel_size=(8, 8, -1),  # H W T
    use_spatial_rope=True,
    class_dropout_prob=0.0,
    force_huggingface=True,
)
vae = dict(
    type="OpenSoraVAE_V1_3",
    from_pretrained="/workspace/Open-Sora/hpcai-tech/OpenSora-VAE-v1.3",
    z_channels=16,
    micro_batch_size=1,
    micro_batch_size_2d=4,
    micro_frame_size=17,
    use_tiled_conv3d=True,
    tile_size=4,
    normalization="video",
    temporal_overlap=True,
    force_huggingface=True,
)
text_encoder = dict(
    type="t5",
    from_pretrained="/workspace/Open-Sora/pretrained_models/DeepFloyd/t5-v1_1-xxl",
    model_max_length=300,
)
scheduler = dict(
    type="rflow",
    use_timestep_transform=True,
    num_sampling_steps=30,
    cfg_scale=7.5,
    scale_image_weight=True,
    initial_image_scale=1.0,
)
image_cfg_scale = 5.0  # cfg, increase to force follow
aes = 7.0
flow = None

i2v inference command:
python scripts/inference_i2v.py \
configs/opensora-v1-3/inference/v2v.py   \
--num-frames 49 \
--resolution 360p \
--aspect-ratio "9:16" \
--cond-type i2v_head \
--use-sdedit True   \
--use-oscillation-guidance-for-image True \
--image-cfg-scale 2.0   \
--use-oscillation-guidance-for-text True \
--cfg-scale 7.5   \
--prompt 'A breathtaking sunrise scene.{"reference_path": "assets/images/condition/wave.png"}'

![Image](https://github.com/user-attachments/assets/db927a12-e279-492e-b02d-82a7044ef070)

**Comment by Gracewx** [comment]
> Same issue on my end as well. Any ideas?
Training from scratch I guess? I haven't tried it.
Maybe you can follow this link to create your docker.
https://blog.csdn.net/zzh516451964zzh/article/details/128992983

**Comment by ZoedAb** [comment]
Same problem
